README.md

```diff
@@ -626,6 +626,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Mike Hatch](https://github.com/mikeshatch)
   - [Milind Mishra](https://github.com/thatbeautifuldream)
   - [Mitanshi Kshatriya](https://github.com/MitanshiKshatriya)
+  - [Mrutunjay Singh](https://github.com/Mrityunjay1)
```

Comment:
Please add your name in line 647, after Mrinaal Arora.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable